flexmeasures/data/models/user.py

```diff
@@ -188,6 +189,7 @@ class User(db.Model, UserMixin, AuthModelMixin):
     username = Column(String(255), unique=True)
     password = Column(String(255))
     last_login_at = Column(DateTime())
+    last_seen_at = db.Column(DateTime())
```

Comment:
Maybe this is a good place for a short comment on the distinction between last_login_at and last_seen_at.

Reply:
`db.` can go.

Reply:
I actually think the names of the columns are pretty self-explanatory. What was unclear when you read them?

Reply:
Hehe, I honestly don't know what the difference is.